rpcs3/rpcs3qt/save_manager_dialog.cpp

```diff
@@ -273,66 +344,18 @@ void save_manager_dialog::OnEntriesRemove()
 		qSort(selection.begin(), selection.end(), qGreater<QModelIndex>());
 		for (QModelIndex index : selection)
 		{
-			int idx_real = m_list->item(index.row(), 0)->data(Qt::UserRole).toInt();
+			int idx_real = m_list->item(index.row(), 1)->data(Qt::UserRole).toInt();
 			fs::remove_all(m_dir + m_save_entries[idx_real].dirName + "/");
 			m_list->removeRow(index.row());
 		}
 	}
 }
 
-// Pop-up a small context-menu, being a replacement for save_data_manage_dialog
-void save_manager_dialog::ShowContextMenu(const QPoint &pos)
+void save_manager_dialog::SetIconSize(int size)
 {
-	bool selectedItems = m_list->selectionModel()->selectedRows().size() > 1;
```

Comment:
Why did you remove the contextmenu?
People are gonna add it back.

Reply:
I don't like having same features in two places, everything in the context menu is accessible from other places in the interface. But I can add it back if that is desired.

Reply:
Most will say yes